salt/modules/purefa.py

```diff
@@ -0,0 +1,1274 @@
+# -*- coding: utf-8 -*-
+
+##
+# Copyright 2017 Pure Storage Inc
```

Comment:
We would prefer that this line be removed.

Reply:
@cachedout I'm going to check with my Legal department to see if we can do this.

Reply:
@cachedout This is the response from our Legal department...

"What's the legal basis for requesting that the copyright marking be removed?  

It is Pure's code and we are required to protect copyright on it. We are not dedicating the code to the public domain, rather we are licensing our copyrighted code via an open-source license.  If remove our copyright markings, it appears like the code is free for all to use."

As there are other copyrights (eg SuSE and personal) scattered throughout the Salt code base we will have to let this stay.

Reply:
@sdodsley normally we avoid these unless they come from the supporting upstream vendor, like it is in this case. Sorry for the confusion, we will get this merged in :)